src/code42cli/cmds/search/extraction.py

```diff
@@ -45,7 +44,7 @@ def _get_alert_details(sdk, alert_summary_list):
 
 
 def create_handlers(
```

Comment:
Instead of changing this implementation, make a new method for `create_server_handlers()` or `create_send_to_handlers()` and create the logger inside there and all you need to do it call `logger.info()` on one of the event dicts, and the behavior is then restored.

Reply:
Done